Dockerfile.artifacts

```diff
@@ -67,6 +67,18 @@ RUN \
 
 
 FROM alpine
+
+ARG GITHUB_SHA
+
+LABEL org.opencontainers.image.authors="Umputun <umputun@gmail.com>" \
```

Comment:
Unless I forgot something, I don't see why it is needed in the artifact container. This one is an intermediate thingy and does not produce any image, but rather the binary archives